packages/sui-studio/src/runtime-mocha/index.js

```diff
@@ -0,0 +1,114 @@
+/* global __BASE_DIR__ */
+import React from 'react'
+import hoistNonReactStatics from 'hoist-non-react-statics'
+
+import SUIContext from '@s-ui/react-context'
+import withContext from '../components/demo/HoC/withContext'
+import {cleanDisplayName} from '../components/demo/utilities'
+import {requireFile, tryRequireContext, tryRequireComponent} from '../components/tryRequire'
+
+const originalKarmaLoader = window.__karma__.loaded
+window.__karma__.loaded = () => {}
+
+const regex = /\.\/(?<categoryName>\w+)\/(?<componentName>\w+)\/(src)+\/(index.js|jsx)$/
```

Comment:
I guess this will break if the mono repo depth is 3 instead of 2, right?

Reply:
yeap
[https://regexr.com/570ub](https://regexr.com/570ub)

Reply:
Well, this really isn't for any monorepo, it's just for the Studios, which we already know are only two levels deep.